.licenserc.yaml

```diff
@@ -0,0 +1,9 @@
+header:
+  license:
+    spdx-id: Apache-2.0
+    copyright-owner: Apache Software Foundation
```

Comment:
The copyright owner is not the asf. Not sure what this line does but it seems wrong.

Reply:
Sorry @jacques-n my bad, should the `copyright-owner` be `Substrait Project` or `Substrait`?

Reply:
Maybe just don't have anything? It doesn't actually show up in the license header so I'm not sure why we need anything here. If you have to have something, Substrait Project sounds good. (We're not in a foundation yet.)

Reply:
Okay, i think we can remove it. I will update it now.